controllers/disruption_controller.go

```diff
@@ -94,7 +94,7 @@ func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	r.log = r.BaseLog.With("disruptionName", req.Name, "disruptionNamespace", req.Namespace)
 
 	// reconcile metrics
-	r.handleMetricSinkError(r.MetricsSink.MetricReconcile())
+	r.handleMetricSinkError(r.MetricsSink.MetricReconcile([]string{"controller", r.MetricsSink.GetSinkName()}))
```

Comment:
If the controller is in the prefix, why also have it as a tag?

Reply:
Yup, no need to have it in the tag, going to remove for all.